ganache-with-protocol/build-ganache.sh

```diff
@@ -0,0 +1,90 @@
+git clone -b streamflow https://github.com/livepeer/protocol.git
+cd protocol
+echo "Setting devenv specific protocol parameters"
+migrations="/protocol/migrations/migrations.config.js"
+sed -i 's/roundLength:.*$/roundLength: 50,/' $migrations
```

Comment:
@adamsoffer These values override some of the values specified in https://github.com/livepeer/protocol/blob/streamflow/migrations/migrations.config.js. I recall you mentioning you tweaked some contract parameters in your subgraph integration tests so I wanted to draw your attention here. Worth noting that it should also be possible to change the parameters in the tests by sending a transaction to update one of the parameters using the first account managed by ganache.

Reply:
Yeah, I changed the round length to a small number which I see is set to 50 here so that should be good. Thanks for the reminder I can also change this parameter in the tests.